website/docs/openmp.md

```diff
@@ -20,15 +20,10 @@ Project configurations.
 
 ### Availability ###
 
-Premake 5.0-beta1 or later. Currently only implemented for Visual Studio 2010+. As a workaround for other toolsets, you can use [buildoptions](buildoptions.md) like this:
-
-```lua
-filter "toolset:not msc*"
-	buildoptions "-fopenmp"
-```
+Premake 5.0-beta1 or later.
```

Comment:
Changed to add beta 2 for gcc/clang.
It is unrelated to IDE, it is toolset side.

Reply:
> It is unrelated to IDE, it is toolset side.

???

Toolset:
https://github.com/premake/premake-core/blob/5c2a1cccd7fc70d691bba0333718e12b2572ee4a/src/tools/msc.lua#L134-L137

IDE:
https://github.com/premake/premake-core/blob/5c2a1cccd7fc70d691bba0333718e12b2572ee4a/modules/vstudio/vs2010_vcxproj.lua#L2217-L2223

Reply:
As for optimization -O2, -O3, and vs need special treatment to "parse" options. (just adding `/openmp` will be in extra build option instead of setting the dedicated widget).
Codelite/Codeblocks/.. parse options to update the associated widget (when appropriate entry has be added), so no code changes is required for IDEs (except vs).

I still think the msvc way we do that is not the best. (`buildoptions "/openmd"` will not show the option on).
But it is irrelevant.